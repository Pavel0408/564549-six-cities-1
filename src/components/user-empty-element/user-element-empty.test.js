import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom";

import {UserEmptyElement} from "./user-empty-element";

describe(`testing UserEmptyElement render`, () => {
  it(`UserEmptyElement is render correctly`, () => {
    const tree = renderer.create(
        <BrowserRouter>
          <UserEmptyElement
            user={null}
          />
        </BrowserRouter>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});


