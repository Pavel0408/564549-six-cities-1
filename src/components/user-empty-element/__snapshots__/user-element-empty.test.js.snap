// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing UserEmptyElement render UserEmptyElement is render correctly 1`] = `
<a
  className="header__nav-link header__nav-link--profile"
  href="/login"
  onClick={[Function]}
>
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />
  <span
    className="header__login"
  >
    Sign in
  </span>
</a>
`;
